### ISSUE TITLE
preact setting

### ISSUE THREAD

**Issue by fingerk28** [Jia-Research-Lab/ReviewKD#11]
Line 222, ReviewKD/CIFAR-100/train.py
`t_features, t_pred = teacher(images, is_feat = True, preact=True)`

Line 76, ReviewKD/CIFAR-100/model/reviewkd.py
`student_features = self.student(x,is_feat=True)`

Why preact is set different value in these two positions?
One is True, the other is False (default value)
thanks!

**Comment by akuxcw** [comment]
Thank you for this question! Actually, we didn't pay much attention to this part. I'm not sure whether the performance will be higher if we align these positions. You are welcome to try it by yourself!

**Comment by Mdahao** [comment]
> Line 222, ReviewKD/CIFAR-100/train.py
> `t_features, t_pred = teacher(images, is_feat = True, preact=True)`
> 
> Line 76, ReviewKD/CIFAR-100/model/reviewkd.py
> `student_features = self.student(x,is_feat=True)`
> 
> Why preact is set different value in these two positions?
> One is True, the other is False (default value)
> thanks!

Hi,  student_features = self.student(x,is_feat=True)..... What are  the role of    'x'   and   'is_feat'  ？，I am a little puzzled.

**Comment by Mdahao** [comment]
> Thank you for this question! Actually, we didn't pay much attention to this part. I'm not sure whether the performance will be higher if we align these positions. You are welcome to try it by yourself!

Hi sir, student_features = self.student(x,is_feat=True)..... What are the role of 'x' and 'is_feat' ？，I am a little puzzled.

**Comment by Mdahao** [comment]
> Thank you for this question! Actually, we didn't pay much attention to this part. I'm not sure whether the performance will be higher if we align these positions. You are welcome to try it by yourself!

Hi sir, student_features = self.student(x,is_feat=True)..... What is the role of  'is_feat' ？，I am a little puzzled.

**Comment by fingerk28** [comment]
> > Thank you for this question! Actually, we didn't pay much attention to this part. I'm not sure whether the performance will be higher if we align these positions. You are welcome to try it by yourself!
> 
> Hi sir, student_features = self.student(x,is_feat=True)..... What is the role of 'is_feat' ？，I am a little puzzled.

If is_feat=True, model will return logits and feature map, otherwise just return logits.

**Comment by Mdahao** [comment]
> > > Thank you for this question! Actually, we didn't pay much attention to this part. I'm not sure whether the performance will be higher if we align these positions. You are welcome to try it by yourself!
> > 
> > 
> > Hi sir, student_features = self.student(x,is_feat=True)..... What is the role of 'is_feat' ？，I am a little puzzled.
> 
> If is_feat=True, model will return logits and feature map, otherwise just return logits.

Thank you for this reply, sir, and one question, is that the location called  "review."？
       82    for features, abf, shape, out_shape in zip(x[1:], self.abfs[1:], self.shapes[1:], self.out_shapes[1:]):
